src/App.tsx

```diff
@@ -28,7 +28,7 @@ import MainRouter from './screens/Router'
 // import { fixtures } from './tests/__mocks__/fixtures'
 
 const App: React.FC = () => {
-  const [isAuth, setIsAuth] = useState(true)
+  const [isAuth, setIsAuth] = useState(false)
```

Comment:
Пока авторизация не готова, лучше оставить как было))

```suggestion
  const [isAuth, setIsAuth] = useState(true)
```

Reply:
поправил